webapp/src/components/App/Singlestudy/explore/Modelization/Map/MapConfig/Layers/UpdateLayerDialog.tsx

```diff
@@ -104,15 +105,8 @@ function UpdateLayerDialog(props: Props) {
             control={control}
             fullWidth
             rules={{
-              required: { value: true, message: t("form.field.required") },
-              validate: (v) => {
-                if (v.trim().length <= 0) {
-                  return false;
-                }
-                if (existingLayers.includes(v.toLowerCase())) {
-                  return `The Layer "${v}" already exists`;
-                }
-              },
+              validate: (v) =>
+                validateString(v, { existingEntries: existingLayers }),
```

Comment:
If you define the same name, you will have an error

Reply:
ok I will test it